service/dps/rest/src/main/java/eu/europeana/cloud/service/dps/rest/TopologyTasksResource.java

```diff
@@ -219,6 +217,43 @@ public void run() {
         return Response.notModified().build();
     }
 
+
+    @POST
+    @Consumes({MediaType.APPLICATION_JSON})
+    @PreAuthorize("hasPermission(#topologyName,'" + TOPOLOGY_PREFIX + "', write)")
+    @Path("{taskId}/cleaner")
+    public void cleanIndexingDataSet(@Suspended final AsyncResponse asyncResponse,
+                                     @PathParam("topologyName") final String topologyName,
+                                     @PathParam("taskId") final String taskId,
+                                     final DataSetCleanerParameters cleanerParameters
+    ) throws AccessDeniedOrTopologyDoesNotExistException, AccessDeniedOrObjectDoesNotExistException {
+        assertContainTopology(topologyName);
+        reportService.checkIfTaskExists(taskId, topologyName);
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    asyncResponse.resume("The request was received successfully");
+                    if (cleanerParameters != null) {
+                        LOGGER.info("cleaning dataset {} based on date: {}", cleanerParameters.getDataSetId(), cleanerParameters.getCleaningDate());
+                        DatasetCleaner datasetCleaner = new DatasetCleaner(cleanerParameters);
+                        datasetCleaner.execute();
+                        LOGGER.info("dataset{} cleaned successfully", cleanerParameters.getDataSetId());
```

Comment:
insert space betwen dataset and {}
now in log it looks like below. 
Maybe messages should always start with capital letter ?

2019-09-25T07:44:03.127Z dps [eu.europeana.cloud.service.dps.metis.indexing.DatasetCleaner] - INFO : Data set removed
2019-09-25T07:44:03.127Z dps [eu.europeana.cloud.service.dps.rest.TopologyTasksResource$2] - INFO : dataset25 cleaned successfully